charts/spv-wallet-stack/charts/spv-wallet/values.yaml

```diff
@@ -111,6 +111,7 @@ config:
       max_open_connections: 0
       # it must be set here at least as empty otherwise we won't be able to override it with env
       password: ""
+      user: postgres
```

Comment:
in configmap it is taken from the postgres chart dependency so it's even not used.